CMakeLists.txt

```diff
@@ -57,9 +57,12 @@ endif()
 
 set(CMAKE_CXX_STANDARD 17)
 
-# Speed up builds on HDDs
+# Speed up builds on HDDs and prevent wearing of SDDs
 if(GNU_GCC OR LLVM_CLANG)
-  add_compile_options(-pipe)
+  option(BUILD_LOW_MEMORY "Store temporary build files on disk on low-memory systems" OFF)
```

Comment:
We should either  reference -pipe in the description or rename to macro to, to not require the user to read the implementation. COMPILE_OPTION_PIPE or such.

Reply:
I think mentioning it in the description suffices.